### PR TITLE
Sponsors

### PR DESCRIPTION
**Pull request by mehakg99**

### CODE REVIEW

File: src/Pages/Sponsors/SponsorCard.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from "./Sponsors.module.css";
+
+const SponsorCard = ({ image, name, desc,index }) => {
+  return (
+    <div
+      className={`d-flex flex-column justify-content-end py-3 align-items-center me-4 mb-4 animate__animated animate__fadeIn ${styles.card} ${index%2==0 ? styles.rightrotate : styles.leftrotate}`}
+    >
+      <div className="h-50 d-flex flex-row justify-content-center m-auto">
+        <img style={{height:"120px",width:"140px"}}
```
Comment (puneetkalsi99) [inline src/Pages/Sponsors/SponsorCard.jsx:10]: static sizing?

File: src/Pages/Sponsors/Sponsors.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+           <div className='m-4' key={sponsor.id}>
+           <h1  className={`text-white text-center main_font text-uppercase animate__animated animate__fadeIn ${styles.heading}`}>{sponsor.name}</h1>
+          <center><div
+          className={`m-4 w-25 ${styles.headingLine}`}
+        >
+          &nbsp;
+        </div></center>
+
+        <div className="d-flex flex-row w-100 flex-wrap justify-content-center">
+          {sponsor.sponsors.map((sponsorchild,index) => {
```
Comment (puneetkalsi99) [inline src/Pages/Sponsors/Sponsors.jsx:47]: camel casing, sponsorChild

Comment (puneetkalsi99) [reply src/Pages/Sponsors/Sponsors.jsx:47]: also no need to write =>{ 
return ()
}
just write => ()

**Review by puneetkalsi99 — COMMENTED** [review]
check the comments

File: src/Pages/Sponsors/Sponsors.jsx (hunk 2 above)
Comment (kannan-gill) [reply src/Pages/Sponsors/Sponsors.jsx:47]: merse bina return ke chala nhi, so i did that

**Pull request closed without merging**